Text Enhancements (#336)
HypixelDev/PublicAPI: Documentation/README.md

# Documentation  
Within this folder, you will find detailed instructions on the various functions the Hypixel Public API allows.

## Date and Time  

> Generally, date-times are stored as Unix epoch times in milliseconds.

Author: Should I add a Unix Epoch time Converter Link for people? (https://www.epochconverter.com/)

Reviewer: I don't think that is necessary. They should be able to find one on their own if they need it.

Author: alright

## Response Format  
Responses are served in JSON format.

## UUID Parameters
Methods which require a UUID in the API ask for a Java `UUID` object, which can be built using `UUID#fromString`. 

**Note:** This method will **only** accept UUIDs in dashed format.

``Correct Format: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``

> ``Incorrect Format: f7c77d999f154a66a87dc4a51ef30d19``

Author: Like this?

Reviewer 2: I suppose this will work

# Documentation  
Within this folder, you will find detailed instructions on the various functions the Hypixel Public API allows.

## Date and Time  

> Generally dates are stored as a Unix Epoch times in milliseconds.

Reviewer 2: ```suggestion
Generally, date-times are stored as Unix epoch times in milliseconds.
```

Responses are served in JSON format.

## UUID Parameters

> UUID's will work the same with or without dashes, but it is recommended to format it without dashes.
> 
> ``Recommended Format: f7c77d999f154a66a87dc4a51ef30d19``
> 
> ``Unrecommended Format: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``
> 

Reviewer 2: ```suggestion
Methods which require a UUID in the API ask for a Java `UUID` object, which can be built using `UUID#fromString`. **Note: ** This method will **only** accept UUIDs in dashed format

``Acceptable: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``
``Wrong: f7c77d999f154a66a87dc4a51ef30d19``
```

Author: Dashes aren't really concidered "wrong"... As said in #326. Also I think it should be a bit-less experienced-sounded for people who are new to the API and have no idea what that means.

Reviewer 2: If they want to use the API they should know what `UUID` objects are, you're being too nice

Author: Okay.

Reviewer 2: The example files feature the use of `UUID#fromString` to initialize the API. Also, I said **no** dashes is wrong. And yes, it's wrong. I updated #326 because UUIDs with no dashes actually do not work at all

Author: So the UUID without dashes will not work?

Reviewer 2: Look at the issue. I provided proof. **Only** a dashed UUID will work

Author: okay

## Where Can I Find My UUID?

> The most efficient way is to go on [NameMC](https://namemc.com/), search up the desired profile, and it will show up under 'Minecraft Profile'. [Click here for a visual](https://prntscr.com/vw3f7d).

Author: Do you think this section should be added?

Reviewer 3: Not sure, but anyway wording it like `Where can I find someone's UUID?` is probably better, also you should remove the ad from screenshot.

Author: Good point lol

> ## SkyBlock items and inventories

Reviewer 2: ```suggestion
## SkyBlock Items and Inventories
```
Title case seems to be the preference in the readme

## UUID Parameters
All uuid parameters are expected to have the UUID without dashes.

``Correct Example: f7c77d999f154a66a87dc4a51ef30d19``

> ``Incorrect Example: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``

Reviewer 3: This should wait for https://github.com/HypixelDev/PublicAPI/issues/326, because dashed uuids work just fine.

Author: I could fix that

``Unrecommended Format: f7c77d99-9f15-4a66-a87d-c4a51ef30d19``

## Where can I find someone's UUID?

> The most efficient way is to go on [NameMC](https://namemc.com/), search up the desired profile, and it will show up under 'Minecraft Profile'. [Click here for a visual](https://prnt.sc/vwbqsd).

Author: How does it look now?

Reviewer 2: Actually, that is still incorrect, #326 
I ran some tests, and it runs out `UUID` objects don't accept UUID Strings without dashes **at all**
